ui/app/src/views/home/HomeView.tsx

```diff
@@ -0,0 +1,148 @@
+import { SearchableDashboards } from './SearchableDashboards';
+import { ImportantDashboard } from './ImportantDashboards';
+
+function HomeView() {
+  // Navigate to the project page if the project has been successfully added
+  const navigate = useNavigate();
+
+  const { data } = useProjectList();
+
+  const projectOptions = useMemo(() => {
+    return (data || []).map((project) => project.metadata.name);
+  }, [data]);
+
+  const handleAddProjectDialogSubmit = useCallback(
+    (entity: ProjectModel) => navigate(`/projects/${entity.metadata.name}`),
+    [navigate]
+  );
+
+  const handleAddDashboardDialogSubmit = useCallback(
+    (dashboardSelector: DashboardSelector) =>
+      navigate(`/projects/${dashboardSelector.project}/dashboards/${dashboardSelector.dashboard}/create`),
+    [navigate]
+  );
+
+  // Open/Close management for dialogs
+  const [isAddProjectDialogOpen, setIsAddProjectDialogOpen] = useState(false);
+  const [isAddDashboardDialogOpen, setIsAddDashboardDialogOpen] = useState(false);
+
+  const handleAddProjectDialogOpen = useCallback(() => {
+    setIsAddProjectDialogOpen(true);
+  }, []);
+  const handleAddProjectDialogClose = useCallback(() => {
+    setIsAddProjectDialogOpen(false);
+  }, []);
+
+  const handleAddDashboardDialogOpen = useCallback(() => {
+    setIsAddDashboardDialogOpen(true);
+  }, []);
+  const handleAddDashboardDialogClose = useCallback(() => {
+    setIsAddDashboardDialogOpen(false);
+  }, []);
+
+  return (
+    <Stack sx={{ width: '100%' }} m={2} gap={2}>
+      <DashboardBreadcrumbs />
+      <Box sx={{ width: '100%' }}>
+        <Stack direction="row" alignItems="center" justifyContent="space-between">
+          <Stack direction="row" alignItems="center" gap={1}>
+            <HomeIcon fontSize={'large'} />
+            <Typography variant="h1">Home</Typography>
+          </Stack>
+          <Stack direction="row" gap={2}>
+            <Button
+              variant="contained"
+              size="small"
+              sx={{ textTransform: 'uppercase' }}
+              onClick={handleAddProjectDialogOpen}
+            >
+              Add Project
+            </Button>
+            <Button
+              variant="contained"
+              size="small"
+              sx={{ textTransform: 'uppercase' }}
+              onClick={handleAddDashboardDialogOpen}
+            >
+              Add Dashboard
+            </Button>
+            <AddProjectDialog
+              open={isAddProjectDialogOpen}
+              onClose={handleAddProjectDialogClose}
+              onSuccess={handleAddProjectDialogSubmit}
+            />
+            <CreateDashboardDialog
+              open={isAddDashboardDialogOpen}
+              projectOptions={projectOptions}
+              onClose={handleAddDashboardDialogClose}
+              onSuccess={handleAddDashboardDialogSubmit}
+            />
+          </Stack>
+        </Stack>
+      </Box>
+      <Grid container spacing={8}>
```

Comment:
What do you think about making each grid into its own component?

```suggestion
      <RecentDashboards />
      <ProjectsAndDashboards />
      <ImportantDashboards/>
```

Reply:
This is great, much easier to read. Thanks!